### PR TITLE
StoreNav bar holds logo & color

### PR DESCRIPTION
**Pull request by MatthewVaccaro**
## Description
Small update - the StoreNav bar doesn't rely on information passed to it via props but on making it's own independent call.
## Type of change
Small Changes
## Checklist
This doesn't complete the trello card
## Link to Trello card
https://trello.com/c/yYU7bonR/79-storefront-nav-bar

### CODE REVIEW

**Comment by MatthewVaccaro** [comment]
[StoreFront Nav Bar](https://trello.com/c/yYU7bonR/79-storefront-nav-bar)

File: src/components/singleProduct/buyerSingleProductView.js [hunk 1]
```diff
@@ -49,7 +48,6 @@ function SingleProductView(props) {
 	);
 
 	const dispatchItem = (item) => {
-		console.log('🎩', item);
```
Comment (Cireimu) [inline src/components/singleProduct/buyerSingleProductView.js:52]: Please remove this console.log

File: src/components/singleProduct/index.js [hunk 2]
```diff
@@ -5,13 +5,12 @@ import SingleProductView from './buyerSingleProductView';
 
 function Single(props) {
 	const productId = props.match.params.id;
-	const [ up, setUp ] = useState(false);
-	const cartContents = useSelector((state) => state.cart);
 	const store = useSelector((state) => state.user);
+	console.log('store', store);
```
Comment (Cireimu) [inline src/components/singleProduct/index.js]: Please remove this console.log

File: src/components/store/StoreNav.js [hunk 3]
```diff
@@ -1,53 +1,84 @@
...
+import { useDispatch, useSelector } from "react-redux";
+import * as creators from "../../state/actionCreators";
+import NoLogo from "../../images/PureRetail_Logo.png";
+import history from "../../history";
+import search_icon from "../../images/search-icon.svg";
+import cart_icon from "../../images/cart-icon.svg";
+import axiosWithAuth from "../Auth/axiosWithAuth";
+
+const StoreNav = props => {
+  console.log(props, "STORE NAV PROPS");
```
Comment (Cireimu) [inline src/components/store/StoreNav.js]: Please remove this console.log

File: src/components/store/StoreNav.js [hunk 4]
```diff
@@ -1,53 +1,84 @@
...
+  console.log(props.store.storeUrl);
```
Comment (Cireimu) [inline src/components/store/StoreNav.js]: Remove this one as well

File: src/components/store/StoreNav.js [hunk 5]
```diff
@@ -1,53 +1,84 @@
...
+
+  const [color, setColor] = useState();
+  const [logo, setLogo] = useState();
+
+  const dispatch = useDispatch();
+  const cartContents = useSelector(state => state.cart);
+  const storeDetails = useSelector(state => state.user.user);
+  const sellerId = localStorage.getItem("storeId");
+  console.log(sellerId);
```
Comment (Cireimu) [inline src/components/store/StoreNav.js]: And this

File: src/components/store/StoreView.js [hunk 6]
```diff
@@ -22,6 +22,9 @@ function StoreView(props) {
 	const inventory = useSelector((state) => state.store);
 	const storeDetails = store.user;
 	const searchString = useSelector((state) => state.search);
+	const localStoreId = localStorage.getItem('sellerId');
+
+	console.log('localStorage', localStoreId);
```
Comment (Cireimu) [inline src/components/store/StoreView.js]: Remove this as well

File: src/components/store/StoreView.js [hunk 7]
```diff
@@ -36,8 +39,9 @@ function StoreView(props) {
 
 	return (
 		<div>
+			{console.log(props)}
```
Comment (Cireimu) [inline src/components/store/StoreView.js]: Remove this one as well

**Review by Cireimu — CHANGES_REQUESTED** [review]
Looks good overall, just need you to remove these console.logs

File: src/components/singleProduct/index.js (hunk 2 above)
Comment (erferguson) [reply src/components/singleProduct/index.js]: updated

File: src/components/store/StoreNav.js (hunk 3 above)
Comment (erferguson) [reply src/components/store/StoreNav.js]: updated

File: src/components/store/StoreNav.js (hunk 4 above)
Comment (erferguson) [reply src/components/store/StoreNav.js]: updated

File: src/components/store/StoreNav.js (hunk 5 above)
Comment (erferguson) [reply src/components/store/StoreNav.js]: updated

File: src/components/store/StoreView.js (hunk 6 above)
Comment (erferguson) [reply src/components/store/StoreView.js]: updated

File: src/components/store/StoreView.js (hunk 7 above)
Comment (erferguson) [reply src/components/store/StoreView.js]: updated

File: src/components/store/StoreNav.js [hunk 8]
```diff
@@ -1,53 +1,81 @@
...
+  const [logo, setLogo] = useState();
+
+  const dispatch = useDispatch();
+  const cartContents = useSelector(state => state.cart);
+  const storeDetails = useSelector(state => state.user.user);
+  const sellerId = localStorage.getItem("storeId");
+  const findRef = window.location.href;
+
+  useEffect(() => {
+    const Url = `https://pure-retail-bg-routes-t3ulmxmy.herokuapp.com/api/store/${sellerId}`;
```
Comment (Cireimu) [inline src/components/store/StoreNav.js]: Could you also please switch this over to our testAPI Heroku URL?

Comment (erferguson) [reply src/components/store/StoreNav.js]: updated.

**Review by Cireimu — APPROVED** [review]

**Pull request merged**